cumulusci/tasks/salesforce/Deploy.py

```diff
@@ -42,29 +42,56 @@ class Deploy(BaseSalesforceMetadataApiTask):
         },
     }
 
+    # ignore all files in ./lwc and all sub-directories of a lwc component (i.e. ./lwc/[lwc Component Name]/.*)
+    _directory_blacklist = re.compile("^\\.\\/lwc(\\/.*?\\/.*)?$")
+
+    def _include_directory(self, root):
+        return not self._directory_blacklist.match(root)
+
+    # ignore all lwc component sub-directories (usually __tests__ if anything)
+    _lwc_component_pattern = re.compile("^\\.\\/lwc\\/.*$")
+
+    def _is_lwc_component(self, root):
+        return self._lwc_component_pattern.match(root)
+
+    # whitelist all files ending in one of the following: .js, .js-meta.xml, .html, .css, .svg
+    _lwc_component_file_extension_whitelist = re.compile(
+        "^.*\\.(js(-meta\\.xml)?|html|css|svg)$"
+    )
+
+    def _include_lwc_component_file(self, f):
+        return self._lwc_component_file_extension_whitelist.match(f)
+
+    def _get_api(self, path=None):
+        if not path:
+            path = self.task_config.options__path
+
+        package_zip = self._get_package_zip(path)
+        self.logger.info("Payload size: {} bytes".format(len(package_zip)))
+        return self.api_class(self, package_zip, purge_on_delete=False)
```

Comment:
Moving this is not a problem (don't change it back), but generally I'd prefer for code cleanup to happen in separate pull requests from substantive changes because it makes it easier to know what to pay attention to in code review.